usmqe/ceph/commands.py

```diff
@@ -33,8 +41,12 @@ def cmd(self, command):
         format_str = "--format {}".format(self._format) if self._format else ""
         timeout_str = "--connect-timeout {}".format(self._timeout) \
                       if self._timeout else ""
-        return "{} {} {} {}".format(
+        cmd = "{} {} {} {}".format(
             self._base_command, timeout_str, format_str, command)
+        # TODO EOL workaround because of https://bugzilla.redhat.com/show_bug.cgi?id=1448057
+        if self._format == 'json':
+            cmd = "{} {}".format(cmd, "; echo ''")
```

Comment:
Please use construction similar to [this](https://github.com/Tendrl/usmqe-tests/blob/master/usmqe/ceph/ceph_cluster.py#L195), which will ensure to return non zero return code in case of failure: 
```
CEPH COMMAND && echo || (echo; false)
```